sdk/eventhub/event-hubs/src/eventHubSender.ts

```diff
@@ -682,13 +606,12 @@ export class EventHubSender extends LinkEntity {
 
           try {
             await defaultLock.acquire(this.senderLock, () => {
-              return this._init();
+              return this._init(
+                this._createSenderOptions(getRetryAttemptTimeoutInMs(options.retryOptions))
```

Comment:
The feedback at https://github.com/Azure/azure-sdk-for-js/pull/4446#discussion_r309416920 to avoid inlining applies here as well